include/platform-aws.h

```diff
@@ -72,7 +72,15 @@ class PlatformAWS : public Platform {
 		return fields ? fields->func_idx : -EIO;
 	}
 
+	// Determine AWS at runtime
+	bool is_aws() const;
```

Comment:
Are you actually determining if you're running in AWS, or is this really "system_is_ec2_instance()?